scale/docs/rest/workspace.rst

```diff
@@ -2,181 +2,328 @@
+| title                   | String            | Optional | The human-readable name of the workspace.                      |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| description             | String            | Optional | An optional description of the workspace.                      |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| base_url                | String            | Optional | The URL prefix used to access all files within the workspace.  |
+|                         |                   |          | This field can be null if the workspace is not web-accessible. |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| is_active               | Boolean           | Optional | Whether the workspace is available for use. Defaults to true.  |
+|                         |                   |          | Becomes false once a workspace is archived.                    |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| json_config             | JSON Object       | Required | JSON description of the configuration for the workspace.       |
+|                         |                   |          | (See :ref:`architecture_workspaces_spec`)                      |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| .. code-block:: javascript                                                                                              |
+|                                                                                                                         |
+|    {                                                                                                                    |
+|        "name": "raw",                                                                                                   |
+|        "title": "Raw Source",                                                                                           |
+|        "description": "Raw Source Workspace",                                                                           |
+|        "base_url": "http://host.com/rs",                                                                                |
+|        "is_active": true,                                                                                               |
+|        "json_config": {                                                                                                 |
+|            "broker": {                                                                                                  |
+|                "type": "host"                                                                                           |
```

Comment:
There's additional information needed for a host broker. We should have a page describing all the available brokers and their broker specific configuration.